DetergentDoseCalculatorTests/DetergentDoseCalculatorTests.csproj

```diff
@@ -16,7 +16,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\..\DetergentDoseCalculator.csproj" />
+    <ProjectReference Include="..\DetergentDoseCalculator\DetergentDoseCalculator.csproj" />
```

Comment:
I moved the calculator into its own directory. Makes it easier to find and keep all code relevant to it in one location. I also moved your tests down one directory.